transports/jms/src/test/java/org/mule/compatibility/transport/jms/integration/JmsTemporaryReplyToTestCase.java

```diff
@@ -49,20 +49,6 @@ public void testReplyEnabledSyncTimeout() throws Exception {
   }
 
   @Test
-  public void testReplyEnabledNonBlocking() throws Exception {
```

Comment:
should this class extend the new abstract class to continue testing the non blocking scenarios?